site/docs/latest/security/authenz.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Encryption and Authentication using TLS
```

Comment:
This should mention "Athenz" rather than TLS, also the filename should be athenz.md